routes/api/contacts.js

```diff
@@ -1,25 +1,96 @@
-const express = require('express')
+const express = require("express");
+const contacts = require("../../models/contacts");
+const Joi = require("joi");
 
-const router = express.Router()
+const contactSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().email().required(),
+  phone: Joi.string().required(),
+});
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const router = express.Router();
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/", async (req, res, next) => {
+  try {
+    const contactsList = await contacts.listContacts();
+    res.status(200).json({
+      message: "success",
+      data: { contactsList },
+    });
+  } catch (error) {
+    console.log(error);
```

Comment:
what will you return in response if code reach `catch` ? Check in postman :)

Reply:
You should return strict message what happened in API.

Reply:
done